docs/www/shadow-indexing.md

```diff
@@ -0,0 +1,401 @@
+---
+title: Data Management
+order: 1
+---
+
+# Data Management 
+
+One of the key aspects of working with Kafka is how you manage your data.
+
+Creating and maintaining backup scripts is always a pain in the neck and although it can improve reliability it doesn't bring value to the customer directly.
+
+This is the type of work that we would like to avoid and Redpanda can help you with that.
+
+Introducing **Shadow Indexing**.
+
+## Shadow Indexing
+
+**Shadow Indexing** is a unique capability in Redpanda that allows you to archive your data to object storage and recreate complete indexes from it, for all your historic data. 
+
+This capability unlocks new features such as replicating topics from one cluster to another, disaster recovery, etc. without burdening your clusters with local storage and compute.
+
+It's all implemented with [minio](https://min.io/) so it should work with any cloud provider. We officially support Amazon's S3 and Google Cloud Storage. 
+
+You just have to configure your cloud credentials and we'll do the hard work for you. 
+
+With **Shadow Indexing**, Redpanda eliminates the need for disaster recovery by integrating with these infinitely scalable storage systems. It will push the old segments onto the cloud and fetch them when clients need them, with almost no risk to data loss.  
+
+On the surface, this sounds like the obvious solution in a cloud native world, but what it does for enterprises is that it unifies the way you access and manage both historical and real-time data without changing a single line of code in your application!
+
+We call this technology Shadow Indexing because it doesn’t keep the historical data on local disks, instead it keeps a location reference of how and where to fetch the data should a Kafka-API client request it. 
+
+What enterprises are the most excited about, is the fact that you can spin up a new Redpanda cluster and replay petabytes of data to retrain your machine learning model without affecting your production cluster or changing any code. 
+
+In a way, this presents a true disaggregation of compute and store for streaming data.
+
+## How it works
+
+When you create a topic with Redpanda, it automatically toggles the flag cloud_storage_enabled to integrate your data with the cloud. If you have your credentials set, then as soon as possible it will be pushed onto the object store of your choosing.
```

Comment:
The `cloud_storage_enabled` is a global flag that enables/disables all shadow indexing workloads on a broker level

To enable shadow indexing you need to have the flag enabled, and when you're creating the topic you need to set two options `x-redpanda-remote-read` and `x-redpanda-remote-write`.

Reply:
So in short, there's not much that happens automatically, right?

Reply:
Yes, this stuff is disabled by default. Should be explicitly enabled by the user.

Reply:
Ok, addressed. Changed for this text
```suggestion
When you're setting up your Redpanda broker, enable your cloud credentials and flags at `redpanda.yaml`. More on that down below. 

Then when you create a topic, set the flags for remote read/write.

If everything is correct, then as soon as possible your data will be pushed onto the object store of your choosing.
```